packages/odyssey-storybook/src/components/odyssey-mui/Accordion/Accordion.stories.tsx

```diff
@@ -123,3 +127,55 @@ export const Multi: StoryObj<AccordionProps> = {
     );
   },
 };
+
+export const Disabled: StoryObj<AccordionProps> = {
+  args: {
+    children: "This is the content of the box.",
+    isDisabled: true,
+  },
+  render: function C(props: AccordionProps) {
+    return (
+      <Accordion label="Title" isDisabled={props.isDisabled}>
+        {props.children}
+      </Accordion>
+    );
+  },
+};
+
+export const Expanded: StoryObj<AccordionProps> = {
+  args: {
+    children: "This is the content of the box.",
+  },
+  render: function C(props: AccordionProps) {
+    const [isExpanded, setIsExpanded] = useState(true);
+    const onChange = useCallback<
+      Exclude<AccordionProps["onChange"], undefined>
```

Comment:
Doesn't `NonNullable` do this exact same thing?

`NonNullable<AccordionProps["onChange"]>`